check-submodules.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Checks out if a submodule has been updated
+SUBMODULES=`git submodule--helper list | awk '{ print $4 }'`
+RUN_CI="no"
+for SUBMODULE in $SUBMODULES
+do
+    git diff --exit-code HEAD^ $SUBMODULE
+    CHANGED=$? # Get git diff exit code
+    if [ $CHANGED -eq 1 ]
+    then
+        RUN_CI="yes"
+        SUBMODULE_NAME=$submodule
+        break
+    fi
+done
+
+# If no submodule has been updated, exit the script
+if [ "$RUN_CI" = "no" ]; then
+    exit 0
+fi
+
+# Install json-diff
+JSON_DIFF_LINK="https://github.com/Luni-4/json-diff/releases/download"
```

Comment:
Nit: why don't you publish it on crates.io, so we can add it to our dev-dependencies?

Reply:
I will publish it as soon as I can. Do you want this script to be transformed in a test?

Reply:
No it's fine this way

Reply:
A test would be nicer, but it isn't worth spending time on it

Reply:
Updated on crates.io